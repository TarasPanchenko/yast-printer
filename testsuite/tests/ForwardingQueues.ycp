{
    include "testsuite.ycp";
    import "Printer";

    Printer::printcap = [
	$["name" : "lp", "uri" : "parallel:/dev/lp0", ],
	$["name" : "lp1", "uri" : "lpd:/dev/lp1", ],
	$["name" : "lp2", "uri" : "lpd:/dev/lp2", ],
	$["name" : "lp3", "uri" : "ipp://cups.suse.de/printers/lp", ],
	];
    DUMP (Printer::ForwardingQueues ());

    Printer::printcap = [
	$["name" : "lp", "uri" : "parallel:/dev/lp0", ],
	$["name" : "lp1", "uri" : "usb:/dev/lp1", ],
	$["name" : "lp2", "uri" : "usb:/dev/lp2", ],
	$["name" : "lp3", "uri" : "ipp://cups.suse.de/printers/lp", ],
	];
    DUMP (Printer::ForwardingQueues ());
}
