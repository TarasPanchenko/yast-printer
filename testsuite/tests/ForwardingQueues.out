Dump	["lp1", "lp2"]
Dump	[]
